setup.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+if [ ! -f ".env" ]; then
+  cp .env.example .env
+
+  echo "Please, fill in VITE_MAPBOX_TOKEN variable in .env file."
+  echo -n "Do you want to do that now? Y/n "
+  read writeMapboxToken
+
+  mapboxToken=""
+  if [[ "${writeMapboxToken}" =~ ["y(?:es)?|1"] ]]; then
+      echo -n "Input your public MapBox key: "
+      read mapboxToken
+  fi
+
+  echo "VITE_MAPBOX_TOKEN='${mapboxToken}'" >> ".env"
+fi
+
+
+docker-compose build --no-cache --pull
+docker-compose up -d
+
+docker-compose exec node npm install
+docker-compose exec node npm run build
+
+docker-compose exec php composer install
+docker-compose exec php php artisan key:generate
+docker-compose up -d --force-recreate php
```

Comment:
I don't think that's the case, because the token is written into a .env file before containers are loaded.
The issue is with Laravel Application Key.

I suspect PHP container caches it somehow, but I need to investigate it.

Reply:
The issue was docker-compose configuration, fixed